src/coreclr/debug/ee/controller.cpp

```diff
@@ -7553,6 +7638,31 @@ void DebuggerStepper::TriggerUnwind(Thread *thread,
     m_reason = unwindReason;
 }
 
+void DebuggerStepper::TriggerMulticastDelegate(BYTE* pDel, INT32 delegateCount)
+{
+    TraceDestination trace;
+    FramePointer fp = LEAF_MOST_FRAME;
+
+    int totalDelegateCount = (int)*(size_t*)((BYTE*)pDel + DelegateObject::GetOffsetOfInvocationCount());
+    if (delegateCount == totalDelegateCount)
+    {
+        PCODE addr = NULL;
```

Comment:
```suggestion
        PCODE addr = (PCODE)NULL;
```
Fix musl build break